src/applications/disability-benefits/686/config/form.js

```diff
@@ -190,6 +193,114 @@ const formConfig = {
         }
       }
     },
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
+          title: 'Veteran Information',
+          path: 'veteran-information',
+          uiSchema: {
+            veteranFullName: _.merge(fullNameUI, {
+              'ui:options': {
+                hideIf: formData => isVeteran(formData)
+              },
+              first: {
+                'ui:title': 'Veteran’s first name',
+                'ui:required': formData => isNotVeteran(formData)
+              },
+              middle: { 'ui:title': 'Veteran’s middle name' },
+              last: {
+                'ui:title': 'Veteran’s last name',
+                'ui:required': formData => isNotVeteran(formData)
+              },
+              suffix: { 'ui:title': 'Veteran’s suffix' }
+            }),
+            veteranSocialSecurityNumber: {
```

Comment:
Typically we do something like this to add things to common components:

```js
veteranSocialSecurityNumber: _.merge(ssnUI, {
  'ui:options': {
    updateSchema: (form, schema) => {
      if (isVeteran(form)) {
        return _.merge(schema, {
          title: 'Your Social Security number'
        });
      }
      return _.merge(schema, {
        title: 'Veteran’s Social Security number'
      });
    }
  },
  'ui:required': formData => !formData['view:noSSN'],
});
```

It makes it less error prone if we adjust the SSN definition later.